RetroBar/Languages/русский.xaml

```diff
@@ -33,15 +33,28 @@
     <s:String x:Key="use_software_rendering">Использовать п_рограммную отрисовку</s:String>
     <s:String x:Key="middle_mouse_to_close">_Закрывать элементы щелчком средней кнопки мыши</s:String>
     <s:String x:Key="add_show_desktop_button">Отображать кнопку "Показать Рабочий Стол"</s:String>
+    <s:String x:Key="enable_auto_hide">А_втоматически скрывать панель задач</s:String>
```

Comment:
This line uses the same alt-code as https://github.com/dremin/RetroBar/pull/763/files#diff-c1dba38147a49b38994e8d6b97b47600e2cfb3ebacbab7652e00b963cc98a2efR30